docs/blog/newsletter-july-2022.rst

```diff
@@ -0,0 +1,118 @@
+
+A good reviewer's guide should include information such as the review methodology, tools and features to use, tips for reviewers, the review scheduling process, coordination with documentarians and other SMEs, and references to additional resources such as style guides. Although it may make sense to add a reviewer's guide to your style guide, they don't have the same purpose: reviews should focus on content rather than form. Likewise, try to promote more substantial technical comments over grammatical reviews.
+
+Your guide might also provide prompts for certain feedback types or questions for reviewers to ask themselves. Prompts such as "Include the 'why' in your comments" encourage reviewers to provide context and background information. General questions such as "What is the author trying to do?", "Did they succeed?", and "Was it worth doing?" can help reviewers get started. You can even add questions for reviewers to answer directly, such as "Does the procedure work as written?" Listing questions means more work when writing the guide, but helps you avoid getting only minimal or subjective feedback.
+
+For more insights to develop a good reviewer's guide, check out these resources:
+
+- `The Art of the Review <https://www.knowledgeowl.com/home/art-of-review>`__ by Swapnil Ogale
+- `Tech Doc: Unbearable lightness of peer-reviewing <https://www.linkedin.com/pulse/tech-doc-unbearable-lightness-peer-reviewing-kristian-klima>`__ by Kristian Klima
+- `conventional:comments <https://conventionalcomments.org/>`__ by Paul Slaughter
+- `08: Technical Editing (chapter in Open Technical Communication) <https://digitalcommons.kennesaw.edu/cgi/viewcontent.cgi?article=1042&context=opentc>`__ by Jonathan Arnett
+
+-----------------------------------------------
+And another thing! Referring to other documents
+-----------------------------------------------
+
+After a workshop on accessibility at Write the Docs Portland, there was a discussion on one particular point of language: how to point the reader to a different heading or topic.
+
+The first option, "see", has plenty of advocates. It's a short, simple word and we generally prefer simple language. It's pretty general and doesn't prescribe how you should be using the linked resource (unlike, for example, "read"). "See" is favored by the 2011 IBM Style Guide and the `Google dev docs style guide <https://developers.google.com/style/word-list?hl=en#letter-s>`__.
+
+But "see" has some downsides: it doesn't apply to folks with visual impairments and its vagueness can make it hard to translate correctly.
+
+The next contender is "refer". Per the `ASD-STE100 Simplified Technical English specification <https://asd-ste100.org/request.html>`_, refer is more correct: it means "to tell a person where to find information/give information," while see is litera_lly "to know with the eyes."
```

Comment:
Does ``ASD-STE100 Simplified Technical English specification <https://asd-ste100.org/request.html>`_` need a second underscore at the end?

Reply:
Not sure if it's necessary, but better to keep it consistent, for sure.